Zend/zend_virtual_cwd.h

```diff
@@ -232,6 +232,7 @@ typedef struct _virtual_cwd_globals {
 	zend_long                   realpath_cache_size;
 	zend_long                   realpath_cache_size_limit;
 	zend_long                   realpath_cache_ttl;
+	zend_ulong                  realpath_cache_clean_index;
```

Comment:
In this case just moving the new member to be the last member in the struct is fine.